README.md

```diff
@@ -58,16 +104,7 @@ Documentation about radspec and the internals of radspec can be found [here](doc
 
 ## Contributing
 
-TBD.
-
-## Aside: Why is natspec unsafe?
-
-Natspec accepts any valid JavaScript. There are multiple reasons this is a bad idea:
```

Comment:
I think this section is still important, but we should make it clear this is about the JS library for natspec (https://github.com/ethereum/natspec.js).